containers/pre_built_workers/php7/Dockerfile

```diff
@@ -87,11 +85,10 @@ RUN gpg2 --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A17031138
 RUN \curl -sSL https://get.rvm.io | bash -s stable
 
 # Install Ruby 2.5
+RUN apt-get --allow-releaseinfo-change update && apt-get install -y procps && apt-get clean
```

Comment:
I see the `procps` was installed earlier in line #80, can I ask is there any specific reason to install it again? Failure on previous installation?

Reply:
Right, every time we use `rvm` it requires to `apt-get update`. Otherwise an error will pop up. ref:https://stackoverflow.com/questions/23650992/ruby-rvm-apt-get-update-error